### PR TITLE
DOCS: create DEVELOPMENT.md docs

### PR DESCRIPTION
**Pull request by v1v**
## What

Moving all the build/packaging tasks from the main README.md docs to a specific one for the development

## Issues

Avoid any misleading in the docs regarding the packaging, see https://github.com/elastic/apm-agent-php/issues/94

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+make -f .ci/Makefile help
+```
+
+_NOTE_: `PHP_VERSION` can be set to a different PHP version.
+
+# Package generation
+
+| :warning: :construction: **WARNING: This stage is still in development!** |
+| --- |
+
```
Comment (v1v) [inline DEVELOPMENT.md:33]: I've included this section to help with the expectations

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# Local development
+
+## Using the docker approach
```
Comment (SergeyKleyman) [inline DEVELOPMENT.md:3]: I would add "Build/package" sub-section and put all the current content under it so it's clear that it's about building/packaging part of development.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -57,12 +57,17 @@ Once your changes are ready to submit for review:
     We ask this of all contributors in order to assure our users of the origin and continuing existence of the code.
     You only need to sign the CLA once.
 
-2. Test your changes
+2. Build and package your changes
+
+    Run the build and package goals to make sure that nothing is broken.
+    See [development](#development) for details.
+
+3. Test your changes
```
Comment (mdelapenya) [inline CONTRIBUTING.md:65]: In the java-agent, we moved this items to the PR template. Does it make sense here? Or we can do it in a follow-up step after talking to the team

Comment (v1v) [reply CONTRIBUTING.md:65]: That's a good point, I'd say let's keep it as is and discuss with the team

Comment (mdelapenya) [reply CONTRIBUTING.md:65]: Agreed!

**Pull request merged**